src/winetricks

```diff
@@ -16895,6 +16895,21 @@ _EOF_
 
 #----------------------------------------------------------------
 
+w_metadata procexp apps \
+    title="Process Explorer" \
+    publisher="Steve P. Miller" \
+    year="2006" \
+    media="download" \
+
+load_procexp()
+{
+    w_download https://download.sysinternals.com/files/ProcessExplorer.zip
```

Comment:
You can use direct links from the `live` site:
https://live.sysinternals.com/procexp.exe
or
https://live.sysinternals.com/procexp64.exe

Reply:
Please add a checksum here.